src/components/Dropdown/docs/index.story.tsx

```diff
@@ -0,0 +1,129 @@
+  description,
+  divider,
+  playground,
+  tab,
+  code as baseCode,
+  tabs,
+  testkit,
+  title,
+} from 'wix-storybook-utils/Sections';
+import { allComponents } from '../../../../stories/utils/allComponents';
+import { Dropdown } from '../';
+import { DROPDOWN_ALIGNMENT } from '../Dropdown';
+import {
+  simpleOptions,
+  numberOptions,
+  optionsWithSections,
+  optionsWithSubtitle,
+} from '../helpers';
+import { Heart } from 'wix-ui-icons-common/dist/src';
+import * as Readme from '../../Dropdown/README.md';
+
+const code = config =>
+  baseCode({ components: allComponents, compact: true, ...config });
+
+const options = [
+  {
+    label: 'base options',
+    value: simpleOptions,
+  },
+  {
+    label: 'w. subtitle',
+    value: optionsWithSubtitle,
+  },
+  {
+    label: 'w. icon',
+    value: simpleOptions.map(option => ({
+      ...option,
+      icon: <Heart />,
+    })),
+  },
+  {
+    label: 'w. icon & subtitle',
+    value: optionsWithSubtitle.map(option => ({
+      ...option,
+      icon: <Heart />,
+    })),
+  },
+  {
+    label: 'w. sections',
+    value: optionsWithSections,
+  },
+  {
+    label: 'numbers',
+    value: numberOptions,
+  },
+];
+
+export default {
+  category: 'Components',
+  storyName: 'Dropdown',
+  component: Dropdown,
+  componentPath: '../Dropdown.tsx',
+  componentProps: () => ({
+    'data-hook': 'storybook-Dropdown',
+    placeholder: 'Placeholder Text',
+    options: simpleOptions,
+  }),
+  exampleProps: {
+    label: '',
+    disabled: false,
+    error: false,
+    alignment: Object.values(DROPDOWN_ALIGNMENT),
+    errorMessage: '',
+    options,
+    initialSelectedId: '',
+    onChange: option => console.log(option),
+  },
+  dataHook: 'storybook-Dropdown',
+  sections: [
+    header(),
+    tabs([
+      tab({
+        title: 'Usage',
+        sections: [
+          description(Readme),
+
+          divider(),
+
+          title('Examples'),
+
+          ...[
+            { title: 'Simple', source: examples.simpleExample },
+            { title: 'Disabled', source: examples.disabledExample },
+            { title: 'w. label', source: examples.withLabelExample },
+            { title: 'Centered', source: examples.alignmentExample },
+            { title: 'Error', source: examples.errorExample },
+            {
+              title: 'Error with message',
+              source: examples.errorWithMessageExample,
+            },
+            {
+              title: 'w. subtitles (max 2 lines)',
+              source: examples.withSubtitlesExample,
+            },
+            { title: 'Section title', source: examples.sectionTitleExample },
+            { title: 'w. icons', source: examples.withIconsExample },
+            {
+              title: 'w. icons + subtitles',
+              source: examples.withIconsAndSubtitlesExample,
+            },
+            { title: 'Min Width', source: examples.minWidthExample },
+            // { title: 'Mobile Example', source: examples.mobileExample },
```

Comment:
I see that we have a mobile example in the `simpleExample`.
So i'm guessing this can be erased?